libs/sdk-ui-kit/styles/scss/header.scss

```diff
@@ -602,3 +608,94 @@ $gd-header-project-width: 230px;
         }
     }
 }
+
+.gd-data-header-menu-section {
+    display: flex;
+    flex: 1 0 auto;
+
+    .icon-sources {
+        border-bottom-right-radius: 0;
+        border-top-right-radius: 0;
+
+        &:hover {
+            border-color: $button-border-hover;
```

Comment:
why don't we use `$gd-color-highlight` here to make it consistent with the styling of Gooddata

Reply:
I have replaced with it thanks